nextflow.config

```diff
@@ -30,23 +30,17 @@ params {
     run_rtgvcfeval             = false
     save_mapped_as_cram        = false
     scatter_count              = 20
-    skip_fastp                 = false
-    skip_gens                  = true
-    skip_germlinecnvcaller     = false
-    skip_haplogrep3            = false
-    skip_peddy                 = false
+    skip_tools                 = null
     skip_me_calling            = false
     skip_me_annotation         = false
     skip_mt_annotation         = false
     skip_repeat_annotation     = false
     skip_repeat_calling        = false
-    skip_smncopynumbercaller   = false
     skip_snv_annotation        = false
```

Comment:
are you planning to add another skip parameter for the remaining params? Or you plan to infer it from samplesheet in future?

Reply:
himm, maybe having them combine into --skip_subworkflows, or adding like --steps "snvcalling, snvannotation,repeat_annotation" might work.. I guess I am into clustering :)

Reply:
That's a good idea. I will do that in a different PR.